webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -3575,702 +3588,9 @@ const AdminDefinition: AdminDefinitionType = {
-                                                    }
-                                                    {groupDeleteCount > 0 &&
-                                                    <li>
-                                                        <FormattedMessage
-                                                            id='admin.ldap.jobExtraInfo.deletedGroups'
-                                                            defaultMessage='Deleted {groupDeleteCount, number} groups.'
-                                                            values={{
-                                                                groupDeleteCount,
-                                                            }}
-                                                        />
-                                                    </li>
-                                                    }
-                                                    {groupMemberDeleteCount > 0 &&
-                                                    <li>
-                                                        <FormattedMessage
-                                                            id='admin.ldap.jobExtraInfo.deletedGroupMembers'
-                                                            defaultMessage='Deleted {groupMemberDeleteCount, number} group members.'
-                                                            values={{
-                                                                groupMemberDeleteCount,
-                                                            }}
-                                                        />
-                                                    </li>
-                                                    }
-                                                    {groupMemberAddCount > 0 &&
-                                                    <li>
-                                                        <FormattedMessage
-                                                            id='admin.ldap.jobExtraInfo.addedGroupMembers'
-                                                            defaultMessage='Added {groupMemberAddCount, number} group members.'
-                                                            values={{
-                                                                groupMemberAddCount,
-                                                            }}
-                                                        />
-                                                    </li>
-                                                    }
-                                                </ul>
-                                            </span>
-                                        );
-                                    },
-                                },
-                            ],
-                        },
-                    ],
+                    id: 'LdapWizard',
+                    component: LDAPWizard,
```

Comment:
the whole definition is just moved down, with no changes. This component will do what the definition was doing.